src/modules/receipt/components/MedicalRecordButton.tsx

```diff
@@ -0,0 +1,56 @@
+import Button from '@/components/atom/button';
+import { useEffect, useState } from 'react';
+import { splunkBookingInstance } from '@/common/services/splunk';
+
+type props = {
+  bookId: string | string[] | undefined;
```

Comment:
bookId is string type

Reply:
I get this error when I type only as string

TS2322: Type  string | string[] | undefined  is not assignable to type  string | undefined 
Type  string[]  is not assignable to type  string 